src/Main/Form/UpdateProposal.js

```diff
@@ -647,12 +648,15 @@ function UpdateProposal(props) {
                   </Form.Group>
                 </Row>
             <div className="d-flex d-row justify-content-end" style={{marginTop:'44px', marginRight:'20px'}}>
-            <Button onClick={closeModal} style={{color:'#70757A', backgroundColor:'#FFFFFF', borderColor:'#70757A', marginRight:'20px'}}>
+            {/* <Button onClick={closeModal} style={{color:'#70757A', backgroundColor:'#FFFFFF', borderColor:'#70757A', marginRight:'20px'}}>
               Cancel
             </Button>
             <Button style={{backgroundColor:PRIMARY_COLOR}} type="submit">
               Update Proposal
-            </Button>
+            </Button> */}
+            
+            <TFButton label="Cancel" variant="secondary" handleClick={closeModal} customStyles={{marginRight: '20px'}} size="small"/>
+              <TFButton label="Update Proposal" type="submit" size="small"/>

```

Comment:
You dont need the type in the props as this cutom button doesnt take any type prop